source/monitoring.txt

```diff
@@ -212,22 +367,34 @@ Source Connector JMX Metrics
 
    * - **records**
      - The total number of records a MongoDB source task passed to the
-       {+kafka-connect+} framework.
+       {+kc+} framework.
 
    * - **records-filtered**
      - The number of records a MongoDB source task passed to the
-       {+kafka-connect+} framework that were then filtered by the framework.
+       {+kc+} framework that were then filtered by the framework.
        A filtered record is not written to Kafka.
 
    * - **records-acknowledged**
      - The total number of records a MongoDB source task passed to the
-       {+kafka-connect+} framework that were then successfully
+       {+kc+} framework that were then successfully
        written to Kafka.
 
    * - **mongodb-bytes-read**
      - The total number of bytes a MongoDB source task read from the
        MongoDB server.
 
+       .. tip:: Average Record size
+
+          You can calculate the average size of the records your sink connector
+          processed over a span of time by performing the following actions:
+
+          #. Determine the change in the value of the ``mongodb-bytes-read``
+             attribute for a span of time.
+          #. Determine the change in the value of the ``records`` attribute
+             for the same span of time you used for the preceding step.
+          #. Divide the change in the value of the ``mongodb-bytes-read`` attribute
+             by the change in the value of the ``records`` attribute.
+
```

Comment:
Issue: [Style guide](https://www.mongodb.com/docs/meta/style-guide/style/callouts/#callout-guidelines) says to "avoid" nesting callouts in tables.

Suggestion: Could it go somewhere else, like at the end of the table?

Reply:
I think this is ok as is. I think moving it outside the table would place it too far away from where it is relevant.

The Kafka docs have a lot of [long tables](https://www.mongodb.com/docs/kafka-connector/current/sink-connector/configuration-properties/all-properties/) with many admonitions. If we can come up with a solution to remove admonitions from tables but make the information close to where it is relevant, I'm open to exploring it, but I think that may be handled best as a seperate ticket.